### DISCUSSION TITLE
How to deeply spoof the browser fingerprint User-Agent?

### DISCUSSION THREAD

**Discussion by yuyiqilin** [cefsharp/CefSharp#5017] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

131.3.10

### Operating System

Windows 11

### Architecture

x64

### .Net Version

.Net 4.8.1

### Implementation

WPF

### Reproduction Steps

How to deeply spoof the browser fingerprint User-Agent? After I modified the User-Agent, the request's User-Agent indeed changed, but the server refused to respond. When I don’t spoof the User-Agent, the response is normal. I think it’s because the front-end code, such as JavaScript, detects the real User-Agent, and since it doesn’t match, the service is rejected. How can I spoof or deceive the JavaScript detection

### Expected behavior

How can I spoof or deceive the JavaScript detection

### Actual behavior

I can not spoof or deceive the JavaScript detection

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
In future please use https://github.com/cefsharp/CefSharp/discussions to ask questions.

There are examples of changing the UserAgent at https://github.com/cefsharp/CefSharp/wiki/General-Usage#useragent
